pages/common/pio-project.md

```diff
@@ -0,0 +1,20 @@
+# pio project
+
+> Tool to manage PlatformIO projects.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/project/index.html>.
+
+- Initialize a new PlatformIO based project (defaults to current directory if `--project-dir` is omitted):
```

Comment:
This should be split into 2 examples. One without `--project-dir`, and another with `--project-dir`.